hd-hd/c++-hd.html.markdown

```diff
@@ -0,0 +1,1190 @@
+---
+भाषा: सी++
```

Comment:
Metadata shouldn't be translated, since the deployment script expects english keys there.
The rest is fine.